src/components/implementation/no_interface/vkernel/vk_server.c

```diff
@@ -0,0 +1,47 @@
+
+#include "vk_types.h"
```

Comment:
Is this a `personal preference` or something like a CSG standard? 
To me it is good to have `""` for local headers because when I look at it, I know that it is specific to this directory and changing anything there will not affect other parts of the system. I'd still go with `""` if it were a `personal choice`.